travis.sh

```diff
@@ -2,34 +2,5 @@
 
 set -uexo pipefail
 
-DIGGER_DIR="../digger"
-DIGGER="../digger/digger"
-
-# set to 64-bit by default
-if [ -z ${MODEL:-} ] ; then
-    MODEL=64
-fi
-
-build_digger() {
-    git clone --recursive https://github.com/CyberShadow/Digger "$DIGGER_DIR"
-    git -C "$DIGGER_DIR" checkout v3.0.0-alpha-5
-    (cd "$DIGGER_DIR" && rdmd --build-only -debug digger)
-}
-
-install_digger() {
-    $DIGGER build --model=$MODEL "master"
-    export PATH=$PWD/work/result/bin:$PATH
-}
-
-if ! [ -d "$DIGGER_DIR" ] ; then
-    build_digger
-fi
-
-install_digger
-
-dmd --version
-rdmd --help | head -n 1
-dub --version
```

Comment:
Why were these 3 lines removed?  They don't seem related to the rest of the PR and I'd have thought it's convenient to have `dmd`, `rdmd` and `dub` report their versions in the build logs.

Reply:
They are automatically added by our Travis integration.